src/Common/src/Interop/Linux/libc/Interop.inotify.cs

```diff
@@ -32,7 +32,7 @@ internal static int inotify_rm_watch(int fd, int wd)
                 }
                 else
                 {
-                    System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
+                    global::System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
```

Comment:
How about we just allow abbreviations for * in Interop.*.SomeExport on the basis of having to specify it at a the call site. With a good 1:1 name in Interop.Libraries, a clarification is just one F12 away:

So we'd have:
Interop.Libraries.SystemNative
Interop.Libraries.SystemSecurityCryptographyNative
Interop.Crypto.GetX509Thumbprint(...)
Interop.Sys.Stat(...)